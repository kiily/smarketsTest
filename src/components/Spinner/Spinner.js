import React from 'react';
import './Spinner.scss';

const spinner = () => {
  return (
    <div className="Spinner-container">
      <span className="spinner-text">Retrieving popular events</span>
      <div className="spinner">
        <div></div>
        <div></div>
        <div></div>
        <div></div>
      </div>
    </div>
  );
};

export default spinner;
