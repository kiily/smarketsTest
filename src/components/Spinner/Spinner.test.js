import React from 'react';
import { create } from 'react-test-renderer';
import Spinner from './Spinner';

describe('Spinner component', () => {
  it('should match the snapshot when rendered', () => {
    const component = create(<Spinner />);
    expect(component.toJSON()).toMatchSnapshot();
  });
});
