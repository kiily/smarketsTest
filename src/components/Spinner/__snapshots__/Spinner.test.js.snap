// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinner component should match the snapshot when rendered 1`] = `
<div
  className="Spinner-container"
>
  <span
    className="spinner-text"
  >
    Retrieving popular events
  </span>
  <div
    className="spinner"
  >
    <div />
    <div />
    <div />
    <div />
  </div>
</div>
`;
